metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorPartitionService.java

```diff
@@ -326,7 +326,7 @@ public PartitionsSaveResponse savePartitions(
             return result;
         } catch (NoSuchObjectException exception) {
             if (exception.getMessage() != null && exception.getMessage().startsWith("Partition doesn't exist")) {
-                throw new PartitionNotFoundException(tableName, null, exception);
+                throw new PartitionNotFoundException(tableName, "", exception);
```

Comment:
@zhljen Note this change as this PartitionNotFoundException expects a partition name field but we were passing null.